install/connections.csv

```diff
@@ -86,48 +86,48 @@ CARD,10,Cairn Hill,Portable,,2010-09-04T23:23:02Z,2010-09-29T00:00:00Z
+CHIT,40,Owenga,,0,2007-12-16T00:00:00Z,9999-01-01T00:00:00Z
+CHIT,41,Owenga,,1,2007-12-16T00:00:00Z,9999-01-01T00:00:00Z
+CHST,40,Charleston Constant Bay,,0,2015-06-25T00:00:00Z,9999-01-01T00:00:00Z
+CHST,41,Charleston Constant Bay,,1,2015-06-25T00:00:00Z,9999-01-01T00:00:00Z
 CKCJ,10,Cranky Creek,Short Period,,2001-12-10T03:28:00Z,2001-12-27T02:46:00Z
 CKHZ,10,Cape Kidnappers,,,2007-08-29T03:00:00Z,9999-01-01T00:00:00Z
 CLAM,10,Clarence,Short Period,,1993-10-18T05:51:00Z,1994-03-04T16:26:00Z
 CLFO,10,Clifton,Portable,,2014-03-14T22:02:00Z,2014-09-12T00:00:00Z
 CMWS,20,Cape Campbell Strong Motion,,,2013-07-24T03:00:25Z,2013-11-19T10:00:00Z
-CMWZ,10,Cape Campbell,,,2003-12-10T19:00:02Z,9999-01-01T00:00:00Z
-CMWZ,20,Cape Campbell,,,2016-12-05T06:30:00Z,9999-01-01T00:00:00Z
+CMWZ,10,Cape Campbell,,0,2003-12-10T19:00:02Z,9999-01-01T00:00:00Z
+CMWZ,20,Cape Campbell,,3,2016-12-05T06:30:00Z,9999-01-01T00:00:00Z
 CMWZ,20,Cape Campbell Strong Motion,,,2013-07-24T03:00:10Z,2013-07-24T03:00:15Z
 CNGZ,10,Carnagh Station,,,2007-08-04T00:01:00Z,9999-01-01T00:00:00Z
 COHF,10,Colonial Head,Short Period,,2003-08-24T01:47:00Z,2003-09-15T15:11:00Z
 CONM,10,Conway River,Short Period,,1993-10-22T23:39:00Z,1994-03-11T13:07:00Z
 CONO,10,Conical Hill Otago,Portable,,2014-03-16T20:20:00Z,2014-09-12T00:00:00Z
-COVZ,10,Chateau Observatory,,,2009-11-01T23:31:00Z,9999-01-01T00:00:00Z
-COVZ,30,Chateau Observatory,,,2010-11-30T22:30:01Z,9999-01-01T00:00:00Z
+COVZ,10,Chateau Observatory,,0,2009-11-01T23:31:00Z,9999-01-01T00:00:00Z
+COVZ,30,Chateau Observatory,,3,2010-11-30T22:30:01Z,9999-01-01T00:00:00Z
 COVZ,30,Chateau Observatory,Acoustic,,2007-03-09T00:01:00Z,2010-11-30T22:30:00Z
 COVZ,31,Chateau Observatory,,,2014-04-01T23:40:00Z,2015-02-23T00:00:00Z
 COVZ,32,Chateau Observatory,EARSS Network,,1998-03-25T22:51:00Z,1998-06-11T09:00:00Z
 COVZ,32,Chateau Observatory,Quanterra Network,,1999-12-01T00:00:00Z,2000-11-28T18:46:00Z
 COVZ,90,Chateau Observatory,Acoustic,,2009-11-15T01:00:00Z,2010-11-30T22:30:00Z
 CPE,10,Cape Palliser,Short Period,,1990-10-05T11:52:00Z,1990-10-07T19:26:00Z
-CPIT,40,Castlepoint,,,2009-09-16T00:00:00Z,9999-01-01T00:00:00Z
-CPIT,41,Castlepoint,,,2009-09-16T00:00:00Z,9999-01-01T00:00:00Z
+CPIT,40,Castlepoint,,0,2009-09-16T00:00:00Z,9999-01-01T00:00:00Z
+CPIT,41,Castlepoint,,1,2009-09-16T00:00:00Z,9999-01-01T00:00:00Z
 CPWZ,10,Castlepoint Station,,,2009-07-07T01:00:00Z,9999-01-01T00:00:00Z
 CREC,10,Crestridge,Short Period,,2001-01-09T11:09:00Z,2001-06-24T11:39:00Z
-CRGZ,10,Rukuhanga Station,,,2016-09-03T23:25:01Z,9999-01-01T00:00:00Z
-CRGZ,20,Rukuhanga Station,,,2016-09-03T23:30:00Z,9999-01-01T00:00:00Z
-CRLZ,10,Canterbury Ring Laser,,,2003-03-12T00:00:01Z,2014-01-09T00:00:00Z
+CRGZ,10,Rukuhanga Station,,0,2016-09-03T23:25:01Z,9999-01-01T00:00:00Z
+CRGZ,20,Rukuhanga Station,,3,2016-09-03T23:30:00Z,9999-01-01T00:00:00Z
+CRLZ,10,Canterbury Ring Laser,,0,2003-03-12T00:00:01Z,2014-01-09T00:00:00Z
```

Comment:
shall this one have "Ring Laser" added as role as well, similarly to the 1996-2003 entry for CRLZ?

Reply:
need context ..  should that be a seismometer (though with fancy  lights?:))